### ISSUE TITLE
Tinker's Not Working on Server

### ISSUE THREAD

**Issue by WinshaAdin** [SlimeKnights/TinkersConstruct#2372]
Hello!

My friend and I are trying to start a MultiCraft server. Tinker's has downloaded and the library appears to be there, but when we go to make something like a crafting table it just continues to give us the original item over and over again. The crafting table looks like it was created, but it can't be moved into inventory. Trying to move it only recreates the original block/item.

**Comment by LovelyFlare** [comment]
You need to put Tinkers' on the server as well...

**Comment by bonii-xx** [comment]
You need to run a Forge minecraft version with all mods on the server.

**Comment by WinshaAdin** [comment]
Yes to forge, most recent version.
Tinker's is on the server - I kinda figured that would be the obvious part. It is the most recent version. This is a 1.10.2 build all across the board.
Bonii, I'm going to try to run forge's jar file as the server default. The last time I tried that the server crashed and my host told me I needed to run the minecraft_server.jar.

Any other suggestions are welcome!
